lib/connectors/memory.js

```diff
@@ -252,6 +252,18 @@ Memory.prototype.save = function save(model, data, options, callback) {
   });
 };
 
+Memory.prototype.replace = function save(model, data, options, callback) {
```

Comment:
AFAICT, this method is not used by `lib/dao.js` at all. What is the intended usage?